conf/default-vars.yaml

```diff
@@ -0,0 +1,35 @@
+vars:
+  #families
+  font-primary: 'sans-serif'
+  font-secondary: 'serif'
+  font-heading: 'serif'
+
+  #sizes
+  font-size-body: 14px
+  font-size-large: 16px
+  font-size-small: 12px
+
+  line-height: 1.1
+  line-height-large: 1.5
+
+  #weights
+  font-light: 300
+  font-regular: 400
+  font-semibold: 600
+  font-bold: 700
+  font-xbold: 800
```

Comment:
was just ported over from the previous and used in the helpers.  I guess we can probably simplify to just 3 sizes